### ISSUE TITLE
amp and mimic of H1 robot train error

### ISSUE THREAD

**Issue by linClubs** [robfiras/loco-mujoco#62]
hi , thank you for this awesome open source project!
I run 
~~~python
python experiment.py
~~~
the terminator show as follow:
~~~python
wandb: Using wandb-core as the SDK backend.  Please refer to https://wandb.me/wandb-core for more information.
wandb: Currently logged in as: lin611 (lin12) to https://api.wandb.ai. Use `wandb login --relogin` to force relogin
wandb: Tracking run with wandb version 0.19.9
wandb: Run data is saved locally in /home/seres/code/loco-mujoco/examples/training_examples/jax_amp/wandb/run-20250422_144717-p0q31pfc
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run jolly-aardvark-2
wandb: ⭐️ View project at https://wandb.ai/xxx/amp
wandb: 🚀 View run at https://wandb.ai/xxx/amp/runs/p0q31pfc
INFO:2025-04-22 14:47:18,599:jax._src.xla_bridge:867: Unable to initialize backend 'tpu': INTERNAL: Failed to open libtpu.so: libtpu.so: cannot open shared object file: No such file or directory
[2025-04-22 14:47:18,599][jax._src.xla_bridge][INFO] - Unable to initialize backend 'tpu': INTERNAL: Failed to open libtpu.so: libtpu.so: cannot open shared object file: No such file or directory
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 8718/8718 [00:01<00:00, 5517.34it/s]
111: RuntimeWarning: New trajectories loaded, which overrides the old ones.
  warnings.warn("New trajectories loaded, which overrides the old ones.", RuntimeWarning)
~~~

I add print function, the code stop at `rngs = [jax.random.PRNGKey(i) for i in range(config.experiment.n_seeds+1)]`,
~~~python
rngs = [jax.random.PRNGKey(i) for i in range(config.experiment.n_seeds+1)]  # create rngs from seed
rng, _rng = rngs[0], jnp.squeeze(jnp.vstack(rngs[1:]))
out = train_fn(_rng)
~~~

how to fit it ? thanks.
